BuildResidentialScheduleFile/measure.rb

```diff
@@ -51,6 +51,11 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setDefaultValue('smooth')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('schedules_column_names', false)
+    arg.setDisplayName('Schedules: Column Names')
+    arg.setDescription('A comma-separated list of the column names to generate. If not provided, defaults to all columns.')
```

Comment:
Link to list of all possible/valid column names?

Reply:
Good idea. I can add the possible column names.